<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectCreator</class>
 <widget class="QMainWindow" name="ProjectCreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Создание проекта</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="createPage">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="createTopLabel">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Создать проект:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="createLine">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="1">
           <widget class="QLineEdit" name="dirPlaceEdit"/>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="prNameEdit"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="prNameLabel">
            <property name="text">
             <string>Имя проекта:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="dirPlaceButton">
            <property name="text">
             <string>Выбор пути</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="dirPlaceLabel">
            <property name="text">
             <string>Расположение:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="exitButton1">
            <property name="text">
             <string>Выход</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="nextButton1">
            <property name="text">
             <string>Далее &gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cancelButton1">
            <property name="text">
             <string>Отмена</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="unitsPage">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="unitsTopLabel">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Единицы измерения:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="unitsLine">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1">
           <widget class="QComboBox" name="dimsComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>мкм</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>мм</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>м</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="freqsLabel">
            <property name="text">
             <string>Частота:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="dimsLabel">
            <property name="text">
             <string>Размеры:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="timeLabel">
            <property name="text">
             <string>Время:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="timeComboBox">
            <item>
             <property name="text">
              <string>нс</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>мс</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>с</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="freqsComboBox">
            <item>
             <property name="text">
              <string>Гц</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>МГц</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ГГц</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="backButton2">
            <property name="text">
             <string>&lt; Назад</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="nextButton2">
            <property name="text">
             <string>Далее &gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cancelButton2">
            <property name="text">
             <string>Отмена</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="solverPage">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="solverTopLabel">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Решение:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="solverLine">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="3">
           <widget class="QLabel" name="freqMinUnitLabel">
            <property name="text">
             <string>Гц</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QLineEdit" name="freqStepLineEdit"/>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="freqMaxEdit"/>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="freqMinEdit"/>
          </item>
          <item row="0" column="2">
           <widget class="QComboBox" name="solverComboBox">
            <item>
             <property name="text">
              <string>MoM</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>FDTD</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="freqStepLabel">
            <property name="text">
             <string>Шаг перебора частоты:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QLabel" name="freqStepUnitLabel">
            <property name="text">
             <string>Гц</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="freqStepRadioButton">
            <property name="text">
             <string>Шаг по частоте</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="freqMinLabel">
            <property name="text">
             <string>Минимальная частота:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="freqMaxLabel">
            <property name="text">
             <string>Максимальная частота:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="freqMaxUnitLabel">
            <property name="text">
             <string>Гц</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="solverTypeLabel">
            <property name="text">
             <string>Тип решателя:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QRadioButton" name="pointsNumberRadioButton">
            <property name="text">
             <string>Число точек</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="pointsNumberLabel">
            <property name="text">
             <string>Число точек:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QLineEdit" name="pointsNumberEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="backButton3">
            <property name="text">
             <string>&lt; Назад</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="nextButton3">
            <property name="text">
             <string>Далее &gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cancelButton3">
            <property name="text">
             <string>Отмена</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="finishPage">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="finishTopLabel">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Завершите создание проекта:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="finishTopLine">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QTextBrowser" name="solverTextBrowser"/>
          </item>
          <item>
           <widget class="Line" name="finishLine1">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="unitsTextBrowser"/>
          </item>
          <item>
           <widget class="Line" name="finishLine2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="freqsTextBrowser"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="backButton4">
            <property name="text">
             <string>&lt; Назад</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="createButton4">
            <property name="text">
             <string>Создать</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cancelButton4">
            <property name="text">
             <string>Отмена</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>509</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
